packages/components/.storybook/middleware.js

```diff
@@ -0,0 +1,8 @@
+const path = require('path');
+
+const pkgPath = path.join(__dirname, '../');
+const getMiddleware = require('../../../presets/react/storybook/middleware');
+
+module.exports = (router) => {
```

Comment:
i'm not familiar with .storybook enough but couldn't we point local storybook server to a configuration folder a few levels above? this dancing around with importing/extending configs would be avoided then?